presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcConnection.java

```diff
@@ -48,8 +62,7 @@ public void setupServer()
             throws Exception
     {
         Logging.initialize();
-        server = new TestingPrestoServer();
-
+        server = new TestingPrestoServer(ImmutableList.of(binder -> newSetBinder(binder, SystemTable.class).addBinding().to(ExtraCredentialsSystemTable.class).in(Scopes.SINGLETON)));
```

Comment:
This line is hard to read due to the length. I tried different ways of wrapping but none seem great:
```java
server = new TestingPrestoServer(ImmutableList.of(binder -> newSetBinder(binder, SystemTable.class)
        .addBinding().to(ExtraCredentialsSystemTable.class).in(Scopes.SINGLETON)));
```
Maybe introduce a variable instead:
```java
Module systemTables = binder -> newSetBinder(binder, SystemTable.class)
        .addBinding().to(ExtraCredentialsSystemTable.class).in(Scopes.SINGLETON);
server = new TestingPrestoServer(ImmutableList.of(systemTables);
```